app/views/welcome/index.html.erb

```diff
@@ -2,7 +2,28 @@
     <%= "Search Performance Explorer" %>
 <% end %>
 
+<% content_for :head do %>
+  <script>
+    $(document).ready(function() {
+      $('input[type="radio"]').click(function() {
+        if($(this).attr('id') == 'info_enhanced') {
+          $('.enhanced-checkbox').show();
+        }
+        else {
+          $('.enhanced-checkbox').hide();
+        }
+      });
+    });
+  </script>
+<% end %>
+
 <% content_for :content do %>
+  <noscript>
+    <div class="noscript-notice">
+      <p>JavaScript is turned off and this site won't function properly</p>
+    </div>
+  </noscript>
```

Comment:
This is an internal app. Do we need to deal with the no JS edge case?

Reply:
I'm not sure if our policy has changed on this, but in the past we have gone out of our way to make internal apps work without javascript (I asked about this last year in relation to content tagger).

Reply:
cool, can do this relatively easily :)

Reply:
Just wondering, does this mean that I shouldn't use JS at all or just that it will work properly without?

Reply:
Ah sorry, I meant working properly. Normally we go for [Progressive Enhancement](https://gdstechnology.blog.gov.uk/2016/09/19/why-we-use-progressive-enhancement-to-build-gov-uk/), even on admin apps.

According to @nickcolley we don't need to be so strict in this case, as long as its only our team that will be using the app. So I'd say we should either accept that all our users will use javascript (David's suggestion) or make it work without javascript. What do you think?

If the latter, I'd suggest tackling the non-javascript case as a new story (unless it's really trivial to get it working now) and we can merge this as is for now.

Reply:
cool, i'll leave it as is for the moment then and change it after this commit goes through, saves the commit getting bigger than it already is